### PR TITLE
Solved Issue: 31909

### PR DESCRIPTION
**Pull request by PrathamSinghSaluja**
Pull Request for Issue # .

### Summary of Changes
Solved Issue: 31909 
Made changes in UI of Add more form of Add Image

### CODE REVIEW

File: templates/system/build_incomplete.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+	<script>window.errorLocale={"en-US":{"language":"English US","header":"Environment Setup Incomplete","text1":"It looks like you are trying to run Joomla! from our git repository. To do so requires you to complete a couple of extra steps first.","help-url-text":"More Details"},"en-GB":{"language":"English GB","header":"Environment Setup Incomplete","text1":"It looks like you are trying to run Joomla! from our git repository. To do so requires you to complete a couple of extra steps first.","help-url-text":"More Details"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
```
Comment (richard67) [inline templates/system/build_incomplete.html:9]: Please revert this unrelated and unintended change.

File: templates/system/fatal-error.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
-	<script>window.errorLocale={"en-GB":{"language":"English GB","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"},"en-US":{"language":"English US","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
+	<script>window.errorLocale={"en-US":{"language":"English US","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"},"en-GB":{"language":"English GB","header":"Sorry, there was a problem we could not recover from.","text1":"The server returned a \"{{statusCode_statusText}}\"","help-url-text":"Help me resolve this"}};</script><!-- Sets the content of the translated text, IMPORTANT DO NOT REMOVE -->
```
Comment (richard67) [inline templates/system/fatal-error.html:9]: Please revert this unrelated and unintended change.

**Comment by richard67** [comment]
@PrathamSinghSaluja The issue actually has been already solved with #32728 . So it you fetch latest 4.0-dev branch of the CMS repository, the issue should be solved. Sorry we have forgotten to close that issue so you've made this PR for nothing.

**Comment by PrathamSinghSaluja** [comment]
Hello Sir,
Actually I was added to the Media Manager group and we got a Pdf file where
our tasks and Homework was written and some issues were also there and this
was one of them. I didn't know that it was resolved. Thankyou for
updating me. I will take care.
Regards
Pratham Singh Saluja

**Pull request closed without merging**